fv3core/stencils/d_sw.py

```diff
@@ -475,7 +638,16 @@ def damp_vertical_wind(w, heat_s, diss_e, dt, column_namelist):
 
 
 @gtstencil()
-def ubke(uc: sd, vc: sd, cosa: sd, rsina: sd, ut: sd, ub: sd, dt4: float, dt5: float):
+def ubke(
```

Comment:
One last possible combination -- ubke looks like it could be combined with the step before it, computing ke from ub and vb, they are even in the same domain.

Reply:
Done!